index.ios.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+import type { Region } from './models/Location';
+import type { Callback } from './models/Callback';
+import debounce from './utils/debounce';
+
+const { RNReverseGeocode } = NativeModules;
+
+const searchForLocations = (
```

Comment:
I decided to keep this method in both files in order to keep platform specific flexibility